specification/src/main/asciidoc/webprofile/Introduction.adoc

```diff
@@ -22,7 +22,7 @@ Besides managing HTTP interactions, most web
 applications have significant requirements in the areas of transaction
 management, security and persistence. Such requirements can be readily
 addressed by technologies that have been part of the Jakarta EE platform
-for quite some time, such as the Jakarta Enterprise Beans 3.x
+for quite some time, such as the Jakarta Enterprise Beans 4.x
```

Comment:
This "3.x" reference was meant to indicate the alternate persistence framework defined by EJB 3 and JPA.  So, in this case, I would leave the "3.x" as-is and not update it to "4.x".